libs/blocks/merch-card/merch-card.js

```diff
@@ -1,161 +1,155 @@
+const checkBoxLabel = (ctas, altCtaMetaData) => {
+  const altCtaRegex = /href=".*"/;
+  if (!altCtaRegex.test(altCtaMetaData[1]?.innerHTML)) return null;
+  const allButtons = ctas.querySelectorAll('a');
+  const originalCtaButton = allButtons[allButtons.length - 1];
+  const altCtaButtonData = altCtaMetaData[1];
+  const altCtaButton = createTag('a', { class: [...originalCtaButton.classList, 'alt-cta', 'button--inactive'].join(' ') }, altCtaButtonData.textContent);
+  originalCtaButton.classList.add('active');
+  decorateButtons(altCtaButton);
+  ctas.appendChild(altCtaButton);
+  return altCtaMetaData[0].textContent;
 };
 
-const addInner = (el, altCta, cardType, merchCard) => {
-  const innerElements = [...el.querySelectorAll('h1, h2, h3, h4, h5, h6, p, ul')];
-  const styles = [...el.classList];
-  const merch = styles.includes('merch-card');
-  const pElement = merch && el.querySelector(':scope > div > div > p:last-of-type');
-  const links = pElement ? pElement.querySelectorAll('a') : el.querySelectorAll('a');
-  const list = el.querySelector('ul');
+const isHeadingTag = (tagName) => /^H[1-5]$/.test(tagName);
+const isParagraphTag = (tagName) => tagName === 'P';
+const isListTag = (tagName) => tagName === 'UL';
 
-  const inner = el.querySelector(':scope > div:not([class])');
-  inner.classList.add(`consonant-${cardType}-inner`);
+const createAndAppendTag = (tagName, attributes, content, parent) => {
+  const newElement = createTag(tagName, attributes, content);
+  parent.append(newElement);
+  return newElement;
+};
+
+const parseContent = (el, altCta, cardType, merchCard) => {
+  const innerElements = [...el.querySelectorAll('h1, h2, h3, h4, h5, h6, p, ul')];
+  const bodySlot = createTag('div', { slot: 'body-xs' });
 
   innerElements.forEach((element) => {
-    if (element.tagName.match(/^H[1-6]$/)) element.classList.add(`consonant-${cardType}-title`);
-    if (element.tagName.match(/^P$/)) element.classList.add(`consonant-${cardType}-description`);
-    if (element.tagName.match(/^UL$/)) {
-      list.classList.add(`consonant-${cardType}-list`);
-      list.querySelectorAll('li').forEach((li) => li.classList.add(`consonant-${cardType}-list-item`));
+    const { tagName } = element;
+    if (isHeadingTag(tagName)) {
+      createAndAppendTag(tagName, { slot: textStyles[tagName] }, element.innerHTML, merchCard);
+      return;
+    } if (isParagraphTag(tagName)) {
+      bodySlot.append(element);
+      return;
     }
-  });
 
-  inner.append(...innerElements);
-  addFooter(links, inner, cardType !== PLANS_CARD);
-  decorateFooter(el, altCta, cardType);
-  merchCard.append(inner);
+    if (isListTag(tagName)) {
+      const lastChildOfBody = bodySlot.lastElementChild;
+      if (lastChildOfBody) {
+        bodySlot.removeChild(lastChildOfBody);
+        element.prepend(lastChildOfBody);
+      }
+      createAndAppendTag('div', { slot: 'list' }, element, merchCard);
+    }
```

Comment:
Could de nest I think
```suggestion
    if (bodySlot.lastElementChild) {
      bodySlot.removeChild(lastChildOfBody);
      element.prepend(lastChildOfBody);
    }
    
    if (isListTag(tagName)) {
      createAndAppendTag('div', { slot: 'list' }, element, merchCard);
    }
```